deployments/terraform/create-s3/variables.tf

```diff
@@ -0,0 +1,4 @@
+variable "bucket_name" {
+  description = "Name of bucket"
```

Comment:
Why not reusing the `terraform/variable.tf` ? here it looks that you need to set the variable in multiple places

Reply:
not the same, this is for the module. The idea of a module is to be reusable so we can not mix them.